django-server/server/api/models.py

```diff
@@ -23,7 +23,7 @@ class Cafeteria(models.Model):
 
     @property
     def occupancy_relative(self):
-        return float(self.occupancy) / float(self.capacity)
+        return min(1.0, float(self.occupancy) / float(self.capacity))
```

Comment:
Is it possible to show 1 to regular users and higher values to admins? This is used in admin stats and would distort the data.

Reply:
It's as hard as sending only relative occupancy for not logged users I think. has to leave it for now as the main purpose was to fix counting

Reply:
Okay, I created #166 for that.